src/libsync/account.h

```diff
@@ -322,6 +322,8 @@ public slots:
 
     void userStatusChanged();
 
+    void userOnlineStatusChanged();
```

Comment:
I would prefer something more like `serverUserStatusChanged` that would be easier for me to understand it and the difference with `userStatusChanged` existing signal